src/skeleton.rs

```diff
@@ -9,6 +9,7 @@ use std::str::FromStr;
 #[derive(Deserialize, Serialize, Debug, Clone, PartialEq, Eq)]
 pub struct Skeleton {
     pub manifests: Vec<Manifest>,
+    pub configs: Vec<Manifest>,
```

Comment:
Should we have a "raw file" struct with a manifest wrapper, or is this abuse of manifest ok?